migrations/applied_migrations/Checkbox.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+echo ""
+echo "Applying migration Checkbox"
+
+echo "Adding routes to conf/app.routes"
+
+echo "" >> ../conf/app.routes
+echo "GET        /checkbox                        controllers.CheckboxController.onPageLoad(mode: Mode = NormalMode)" >> ../conf/app.routes
+echo "POST       /checkbox                        controllers.CheckboxController.onSubmit(mode: Mode = NormalMode)" >> ../conf/app.routes
+
+echo "GET        /changeCheckbox                  controllers.CheckboxController.onPageLoad(mode: Mode = CheckMode)" >> ../conf/app.routes
+echo "POST       /changeCheckbox                  controllers.CheckboxController.onSubmit(mode: Mode = CheckMode)" >> ../conf/app.routes
+
+echo "Adding messages to conf.messages"
+echo "" >> ../conf/messages.en
+echo "checkbox.title = checkbox" >> ../conf/messages.en
+echo "checkbox.heading = checkbox" >> ../conf/messages.en
+echo "checkbox.option1 = option1" >> ../conf/messages.en
+echo "checkbox.option2 = option2" >> ../conf/messages.en
+echo "checkbox.checkYourAnswersLabel = checkbox" >> ../conf/messages.en
+echo "checkbox.error.required = Select checkbox" >> ../conf/messages.en
```

Comment:
These don't quite match the model you generated (e.g. you're missing options 3 and 4)